Wikipedia/Code/TalkPageViewController.swift

```diff
@@ -35,9 +36,33 @@ class TalkPageViewController: ViewController {
 
         let rightBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "ellipsis.circle"), style: .plain, target: nil, action: nil)
         navigationItem.rightBarButtonItem = rightBarButtonItem
-
+        
         talkPageView.collectionView.dataSource = self
         talkPageView.collectionView.delegate = self
+
+        setupHeaderView()
+    }
+
+    private func setupHeaderView() {
+        let headerView = TalkPageHeaderView()
+        self.headerView = headerView
+
+        headerView.configure(viewModel: viewModel)
+        navigationBar.isBarHidingEnabled = false
+        navigationBar.isUnderBarViewHidingEnabled = true
+        navigationBar.allowsUnderbarHitsFallThrough = true
```

Comment:
Looks like you'll still want to override `override func point(inside point: CGPoint, with event: UIEvent?) -> Bool {` in the `TalkPageHeaderView`, and return true if they tapped inside anything tappable (I think just the coffee roll Read More button?), and otherwise return false.

You can reference the `OldTalkPageHeaderView` to see what we did there.

Reply:
Great catch - thanks! Updated.